src/mavedb/routers/experiments.py

```diff
@@ -43,45 +43,6 @@
 )
 
 
-# TODO: Rewrite this function.
```

Comment:
Lets retain this function-- I think it still has utility for just listing all experiments.

Reply:
Sure. I'll modify it so that we don't need to send the weird q to it.

Reply:
I think it is ok to just leave it as is for this PR so we can get it released. We can just handle it at some point later-- it isn't hurting anything right now.